wezterm-ssh/src/pty.rs

```diff
@@ -275,4 +292,18 @@ impl crate::session::SessionInner {
         }
         Ok(())
     }
+
+    pub fn channel_eof_and_close(
```

Comment:
Can we rename this to `send_eof_and_close`?
It feels easier to intuit what actions are being taken to me!

Reply:
I'm actually going to remove this one as this was a quick hack to see if it fixed my problem.